recipes/devito-icc-intelmpi/shipyard-config/jobs.yaml

```diff
@@ -0,0 +1,31 @@
+job_specifications:
+- id: bench-icc-intelmp
+  shared_data_volumes:
+  - azureblob_vol
+  tasks:
+  - docker_image: gbisbas/devitoicc:v0.4
+    environment_variables:
+      DEVITO_ARCH: intel
+      OMP_NUM_THREADS: 8
+      DEVITO_MPI: 1
+      DEVITO_OPENMP: 1
+      KMP_AFFINITY: verbose,scatter,granularity=fine
+      DEVITO_LOGGING: 'DEBUG'
+      LC_ALL: 'C.UTF-8'
+      LANG: 'C.UTF-8'
+      I_MPI_DEBUG: '2'
+      I_MPI_FABRICS: 'ofi'
+      I_MPI_PIN: 1
+      I_MPI_PIN_DOMAIN: socket
+      I_MPI_DAPL_PROVIDER: ofa-v2-ib0
+      I_MPI_DYNAMIC_CONNECTION: 0
+      I_MPI_DAPL_TRANSLATION_CACHE: 0
```

Comment:
From what I seen and tried, most of these IMPI env variables are overwrtitten by batch-shipyard to match the supported version on the host.

Reply:
ok, good to know.